### DISCUSSION TITLE
Optical Support Beta Release 1.1-2022-07-27

### DISCUSSION THREAD

**Discussion by androda** [androda/F4_BlueSCSI#4] [Announcements]
This is the place to talk about the new Beta release which includes optical drive support.

**Comment by sclements1** [comment]
Working with the Optical support beta on an F4 - A/UX ISO image appears to mount, but hangs on installation. The only relevant hint I can find is at: https://www.aux-penelope.com/hardware.htm
_I suspect the operating system needs a cd-rom drive with a 512 byte sector size (many newer drives use a sector size of 2048 bytes) or a drive that responds to the "MODE SELECT" command._

I'd also like to know if you can change the SCSI Vendor Config for optical drives like you can for SCSI disks detailed at:
https://github.com/erichelgeson/BlueSCSI/wiki/SCSI-Vendor-Config

**Reply by mynameistroy** [reply]
I've got a fix for this problem in my PR for the mainline BlueSCSI. I suspect Androda will get it merged into the F4 code soon and likely publish another firmware.

Thanks for the info!

**Comment by sclements1** [comment]
Toast image of JP Screensaver does not mount from F4. .toast image downloaded from: https://macintoshgarden.org/apps/jurassic-park-the-screen-saver
Seemingly detected properly according to the LOG file, but volume never mounts. Burned same toast image to physical media and it worked in a CD-ROM drive.

**Reply by androda** [reply]
Thank you for reporting these issues, I'll look into them.

**Reply by androda** [reply]
It would be helpful to know what Mac you're testing on and the operating system version, to see if I can replicate the issue with the screensaver.

**Reply by sclements1** [reply]
These are on a Quadra 650, and it was a fresh install of both System 7.5.3 and MacOS 8.1. Happy to try any other config or variation.
  -Sam
